src/activitylog/activitylog.css

```diff
@@ -1 +1,115 @@
-/* activitylog.css */
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  width: 100%;
+}
+
+#showActivityLog,
+#saveLogBtn {
+  margin: 20px 0;
+}
+
+.activity-log-wrapper {
+  position: relative;
+}
+
+.show-log-detail-wrapper {
+  position: absolute;
+  top: 0;
+  height: 500px;
+  right: 0;
+  width: 40%;
+  display: none;
```

Comment:
It seems that you're using `display: none` with the intention of hiding the wrapper by default, combined with assigning to `.style.display` to toggle its visibility.

Instead of defaulting to `display:none` here, set the `hidden` attribute in HTML, and toggle the `hidden` attribute / property in JavaScript.

- https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden